requirements/requirements-python3.7.txt

```diff
@@ -291,7 +291,7 @@ rsa==4.0
 s3transfer==0.3.3
 sasl==0.2.1
 semver==2.9.1
-sendgrid==6.2.1
+sendgrid==6.2.2
```

Comment:
@potiuk Is it expected that we see changes like this from non-master/cron changes? It's not the end of the world, but it makes it harder to see what caused a version to get upgraded if random PRs update these deps.